packages/salesforcedx-vscode-apex-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -771,18 +771,18 @@ export async function sfdxCreateCheckpoints() {
   // The status message isn't changing, call to localize it once and use the localized string in the
   // progress report.
   const localizedProgressMessage = nls.localize(
-    'sfdx_create_checkpoints_start'
+    'sfdx_update_checkpoints_in_org'
   );
   // Wrap everything in a try/finally to ensure creatingCheckpoints gets set to false
   try {
+    writeToDebuggerOutputWindow(`Starting ${localizedProgressMessage}`);
```

Comment:
The core extension uses https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/messages/i18n.ts#L20. Can you add yours to i18n.ts also?

Reply:
It will be done.